ci/minimum_supported_agent_version_test.go

```diff
@@ -0,0 +1,104 @@
+//go:build ci
+
+package ci
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"path"
+	"strings"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/ops-agent/integration_test/metadata"
+	"gopkg.in/yaml.v2"
+)
+
+var (
+	added    = flag.String("added", "", "Added files")
+	modified = flag.String("modified", "", "Modified files")
+	labels   = flag.String("labels", "", "Pull request labels")
+	baseDir  = flag.String("base-dir", "", "Directory path for the base of the pull request")
+)
+var basePath = os.Getenv("GITHUB_WORKSPACE")
+
+func TestMinimumSupportedAgentVersionIsModified(t *testing.T) {
+	if basePath == "" {
+		t.Fatal("Must be run within git action")
+	}
+
+	if strings.Contains(*labels, "release") {
```

Comment:
Is this the check for whether the stackdriver-instrumentation-release bot is making the change? I'm not familiar with what the labels are populated with but can we make this check more strict? Checking for the bot's github username or failing that, adding some comments here about what the labels are expected to have.

Reply:
A "release" label will be added to the PR, the flag `labels` take in all the labels for the PR delimited by spaces.The stackdriver-instrumentation-release bot has been configured to automatically add the label. That being said any user who needs to edit the `minimum_supported_agent_version` would need to add the label manually to be able to do any required fixes.

Reply:
I see that makes sense to me now. Can we add this as a comment as well, mentioning that the bot will configure this label and maintainers can use the label if they want to fix up `minimum_supported_agent_version` in case of an error

Reply:
Changed in: [63bfb6b](https://github.com/GoogleCloudPlatform/ops-agent/pull/775/commits/63bfb6bfccdaef27380cd30c086a1590ea925d47)